contexts/utopia-natcert/v1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "@context": {
+    "@protected": true,
+    "confidenceMethod": "https://examples.vcplayground.org/contexts/utopia-natcert/vocab#confidenceMethod",
+
+    "ConfidenceMethod": "https://examples.vcplayground.org/contexts/utopia-natcert/vocab#ConfidenceMethod",
```

Comment:
This exists in the "Base Class" table at https://w3c.github.io/vc-data-model/#vocabularies, but I'm not sure how it's supposed to be defined.

Reply:
Fixed in c426eb7